private_cloud.go

```diff
@@ -55,21 +55,48 @@ type PrivateCloudStorageMetric struct {
 	Storage BasicMetric `json:"STORAGE"`
 }
 
+type PrivateCloudMetricsDataTrafficOptions struct {
```

Comment:
This `Granularity` / `Aggregation` / `From` / `To` combination is found many times across the SDK.
I'd suggest making this a single `MetricsOptions` struct (just like we have the `PaginationOptions`)